### DISCUSSION TITLE
Flip Tool

### DISCUSSION THREAD

**Discussion by infojes** [SVG-Edit/svgedit#691] [New feature request]
Would anyone have a flip extension that work with version 2.6+?  The version online now the image reverts back to it's original self.

Thanks

**Comment by infojes** [comment]
We are willing to pay for this feature.

**Comment by brettz9** [comment]
Not sure if you were following or contributing at https://groups.google.com/forum/#!topic/svg-edit/Ol1SIqUi7Io , and it sounds like you may already be aware of it, but in case not, someone had some old code there. I've asked there to confirm licensing in case someone might adapt it to work with the current version.
